From: Matthijs Velsink <mvelsink@gnome.org>
Date: Tue, 13 Feb 2024 10:52:31 +0100
Subject: po: Fix POTFILES.in for the "Remote Login" move

Commit c0c357c3 (!2211) moved the "Remote Login" code to the System
Panel, but did not change the POTFILES.in entries.

This fixes that.
---
 po/POTFILES.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 7c7d87e..0c4cbb5 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -191,7 +191,6 @@ panels/sharing/cc-sharing-networks.ui
 panels/sharing/cc-sharing-panel.c
 panels/sharing/cc-sharing-panel.ui
 panels/sharing/gnome-sharing-panel.desktop.in
-panels/sharing/org.gnome.controlcenter.remote-login-helper.policy.in.in
 panels/sound/cc-alert-chooser-window.c
 panels/sound/cc-alert-chooser-window.ui
 panels/sound/cc-balance-slider.ui
@@ -223,6 +222,9 @@ panels/system/region/cc-region-page.c
 panels/system/region/cc-region-page.ui
 panels/system/remote-desktop/cc-remote-desktop-page.c
 panels/system/remote-desktop/cc-remote-desktop-page.ui
+panels/system/remote-login/cc-remote-login-page.c
+panels/system/remote-login/cc-remote-login-page.ui
+panels/system/remote-login/org.gnome.controlcenter.remote-login-helper.policy.in.in
 panels/system/users/cc-add-user-dialog.c
 panels/system/users/cc-add-user-dialog.ui
 panels/system/users/cc-avatar-chooser.c
